### PR TITLE
Add MAINTAINERS.md and ADOPTERS.md

### PR DESCRIPTION
**Pull request by cgwalters**
These files help clarify/formalize some information. In particular I'd like to look at tools that auto-sync from MAINTAINERS to the github permissions so that we can maintain that in a git-ops fashion.

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+The underlying ostree project is over 13 years old (as of 2024). This project also relates
+to [rpm-ostree](https://github.com/coreos/rpm-ostree/) which has existed a really long time.
+
+Not every one of these projects uses bootc directly today, but a toplevel goal of bootc
+is to be the successor to ostree, and it is our aim to seamlessly carry forward these users.
+
+| Type | Name | Since | Website | Use-Case |
+|:-|:-|:-|:-|:-|
+| Vendor | Endless | 2014 | [link](https://www.endlessos.org/os) | A Completely Free, User-Friendly Operating System Packed with Educational Tools, Games, and More.
+| Vendor | Apertis | 2020 | [link](https://apertis.org) | Collaborative OS platform for products
```
[the remark this answers is not in the document]
Comment (cgwalters) [reply ADOPTERS.md:25]: Thanks, I decided to go with no period (it's implicitly enclosed by the table) and sorting by time

File: ADOPTERS.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+| Vendor | UBlue   | 2023 | [link](https://universal-blue.org/) | The reliability of a Chromebook, but with the flexibility and power of a traditional Linux desktop. 
+| Vendor | Fedora Project | 2021 | [link](https://fedoraproject.org/atomic-desktops/) | An atomic desktop operating system aimed at good support for container-focused workflows.
+| Vendor | Red Hat | 2015 | [link](https://redhat.com) | Image Based Linux
+
+### Adopter Types
+
+**End-user**: The organization runs bootc in production in some way.
+
+**Integration**: The organization has a product that integrates with bootc, but does not contain bootc.
```
[the remark this answers is not in the document]
Comment (cgwalters) [reply ADOPTERS.md:34]: Hmm could be but I was basing this on e.g. https://github.com/kubevirt/kubevirt/blob/main/ADOPTERS.md

[the remark this answers is not in the document]
Comment (djach7) [reply ADOPTERS.md:34]: Ah, that makes sense, I thought it was odd how the verbiage switches up. I will approve momentarily

**Review by djach7 — APPROVED** [review]
LGTM

**Pull request merged**